hip-more-cultural-names/common/landed_titles/0_HIP_MoreCulturalNames.txt

```diff
@@ -18868,10 +18888,21 @@ e_carpathia = {
                 b_skradin = {
                 }
                 b_biograd_na_moru = {
+                    hungarian = "Tengerfeh�rv�r"
+                    szekely = "Tengerfeh�rv�r" # Historical? Copied from Hungarian
+                    spanish = "Zaravecchia"
                 }
                 b_vrana = {
+                    italian = "Aurana"
+                    hungarian = "Vr�na"
+                    szekely = "Vr�na" # Historical? Copied from Hungarian
                 }
                 b_bribir = {
+                    bosnian = "Sidraga"
```

Comment:
sidraga was the county not the town bribir so those are false

Reply:
thanks